### PR TITLE
Make updateInterval available to configure via driver config

### PR DESCRIPTION
**Pull request by limpasha**
I've noticed it's impossible to specify `updateInterval` param for updaters, so this PR is going to provide the ability.

I've tested it on my dev k8s-cluster.

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -1,3 +1,3 @@
 package main
 
-var version = "0.3.0"
+var version = "0.3.1"
```
Comment (limpasha) [inline version.go]: Oh, now I'm wondering, isn't it "CSI Spec Compatibility" version instead of driver's itself version?

File: node.go [hunk 2]
```diff
@@ -241,15 +242,17 @@ func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpu
 }
 
 func (ns *nodeServer) runUpdater(volumeId string, state updaterState, restore bool) error {
-	log.Info().Str("volume_id", volumeId).Msg("starting updater")
+	log.Info().Str("volume_id", volumeId).Str("updateInterval ", state.UpdateInterval.String()).Msg("starting updater")
```
Comment (aleksey-mashanov) [inline node.go]: An extra space after `updateInterval` in a log field name should be removed.

File: version.go (hunk 1 above)
Comment (aleksey-mashanov) [reply version.go]: It is the version of the driver itself.
You add a new functionality not performing a fix, so 0.4.0 will be better.

File: node.go (hunk 2 above)
Comment (aleksey-mashanov) [reply node.go]: `zerolog` has `Dur()` method to log `time.Duration` values.

File: README.md [hunk 3]
```diff
@@ -46,4 +46,5 @@ See [example manifest (dynamic)](./example-dynamic.yaml) for more details.
   * `csi.storage.k8s.io/node-stage-secret-name` - a name of a secret containing `username` and `password` used to authenticate in *onlineconf-admin*. Can contain template variables `${pvc.name}`, `${pvc.namespace}`, `${pv.name}` and `${pvc.annotations['<ANNOTATION_KEY>']}`, see [Kubernetes CSI docs](https://kubernetes-csi.github.io/docs/secrets-and-credentials-storage-class.html#node-stage-secret) for more information. Recommended value is `${pvc.name}`.
   * `csi.storage.k8s.io/node-stage-secret-namespace` - a namespace of this secret. Can contain template variables `${pvc.namespace}` and `${pv.name}`. Recommended value is `${pvc.namespace}`.
   * `uri` - URI of *onlineconf-admin* instance
+  * `updateInterval` - polling interval for requests to *onlineconf-admin* instance (default: "10s")
```
Comment (aleksey-mashanov) [inline README.md:50]: What about static provisioning? Should the documentation be fixed or `updateInterval` does not work with PV?

Comment (limpasha) [reply README.md:50]: It definitely does work with PV, moreover I tested this patch only with PV. Fixed.

It looks like nothing prevents it from being unable to work also with dynamic provisioning, so I leave this line here.

File: version.go (hunk 1 above)
Comment (limpasha) [reply version.go]: Thanks. Changed version to "0.4.0".

File: node.go (hunk 2 above)
Comment (limpasha) [reply node.go]: Thanks for noticing! Fixed this.

**Pull request merged**